src/test/java/org/ambraproject/wombat/controller/ControllerTest.java

```diff
@@ -35,15 +35,17 @@
 import org.ambraproject.wombat.config.SpringMvcConfiguration;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.context.annotation.Bean;
+import org.springframework.test.context.junit4.AbstractJUnit4SpringContextTests;
 import org.springframework.test.context.web.WebAppConfiguration;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.web.context.WebApplicationContext;
-import org.testng.annotations.BeforeMethod;
+import org.junit.Before;
 
 @WebAppConfiguration
 @ContextConfiguration(classes={ControllerTestConfiguration.class, SpringMvcConfiguration.class})
-public class ControllerTest extends AbstractTestNGSpringContextTests {
+public abstract class ControllerTest extends AbstractJUnit4SpringContextTests {
```

Comment:
Do you recall if `abstract` forced by junit or just good expressiveness on your part?

Reply:
It is forced by junit - otherwise junit claims that there are no tests on the class.